events/tests.py

```diff
@@ -0,0 +1,55 @@
+from django.core.exceptions import ValidationError
+from django.utils import timezone
+from .models import Event
+from datetime import datetime
+
+import pytest
+
+
+DATE_TIME_EARLY = datetime(2022, 3, 24, 12, 12, 12, 0, tzinfo=timezone.utc)
+DATE_TIME_LATER = datetime(2022, 3, 24, 14, 12, 12, 0, tzinfo=timezone.utc)
+TITLE = 'new_title'
+
+
+@pytest.fixture
+def new_event():
+    return Event(title='new_title', location='new_location', description='new_description',
```

Comment:
please use the constant from line # 11
and create the same for location and description
```suggestion
    return Event(title=TITLE, location='new_location', description='new_description',
```

Reply:
done